src/map/utils/synthutils.cpp

```diff
@@ -235,7 +235,7 @@ uint8 calcSynthResult(CCharEntity* PChar)
     double chance  = 0;
     double random = tpzrand::GetRandomNumber(1.);
 
-    for(uint8 skillID = 49; skillID < 57; ++skillID)
+    for (uint8 skillID = SKILL_WOODWORKING; skillID <= SKILL_COOKING; ++skillID)
```

Comment:
I like it. :+1:

Reply:
that's better, begone magic numbers. (for some reason I'm getting flashbacks about this)